src/extension.ts

```diff
@@ -105,6 +105,21 @@ export function activate(context: ExtensionContext) {
 							lastStatus = item.text;
 							resolve();
 							commands.registerCommand(Commands.START_DEBUG_SESSION, async (config) => {
+								if (!config.request) { // if 'request' is missing interpret this as a missing launch.json
+									config.type = 'java';
+									config.name = 'Java Debug';
+									config.request = 'launch';
+									const editor = vscode.window.activeTextEditor;
+									if (editor && editor.document.languageId === 'java') {
+										const fullpath = editor.document.fileName;
+										const cur = path.parse(fullpath).dir;
```

Comment:
use `path.dirname(path)` to get dir name.
https://nodejs.org/api/path.html#path_path_dirname_path